web/index.html

```diff
@@ -676,21 +819,26 @@ <h3>Used in</h3>
                 });
 
                 document.addEventListener("wheel", (event) => {
+                    this.isScrolling = true;
                     window.lastScrollTime ??= Date.now();
                     if (this.currentElement && window.lastScrollTime + 200 < Date.now()) {
-                        if (event.deltaX > 20) {
+                        if (event.deltaX > 30 && event.deltaY < 30) {
                             this.slideRight()
                         }
-                        if (event.deltaX < -20) {
+                        if (event.deltaX < -30 && event.deltaY < 30) {
                             this.slideLeft()
                         }
                         window.lastScrollTime = Date.now();
                     }
                 });
 
+                // Preload initial chunks
+                for (let i = 0; i < 10; i++) {
+                    await this.loadChunk(i);
```

Comment:
Preloads the initial chunks async. This allows for them to load one at a time in a different thread without taking any CPU time. You can technically remove the await and have them all load at the same time. I got nearly 100 to load in the same time it currently takes 1, but then the handling functions that are running in the main thread start to get overworked and those can cause slowdowns. Since most of the data is going to be loaded by hover events anyways and not these load chunks, it's better to just load these ones one at a time in the background and you get the best of both worlds.